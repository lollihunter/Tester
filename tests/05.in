20 20 10
1 1
10 1
14 17
2 2
2 13
4 18
18 15
18 1
11 19
20 20
