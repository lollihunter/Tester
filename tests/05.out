8
1 4 5 6 9 3 7 10 
